frontend/packages/metal3-plugin/src/components/dashboard/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { StatusGroupMapper } from '@console/internal/components/dashboard/inventory-card/inventory-item';
+import { InventoryStatusGroup } from '@console/internal/components/dashboard/inventory-card/status-group';
+
+import { getSimpleHostStatus } from '../../utils/host-status';
+import {
+  HOST_ERROR_STATES,
+  HOST_PROGRESS_STATES,
+  HOST_PROVISIONING_STATES,
+  HOST_REGISTERING_STATES,
+  HOST_SUCCESS_STATES,
+} from '../../constants';
+
+const BMH_STATUS_GROUP_MAPPER = {
+  [InventoryStatusGroup.OK]: [...HOST_SUCCESS_STATES],
+  [InventoryStatusGroup.PROGRESS]: [
+    ...HOST_PROGRESS_STATES,
+    ...HOST_PROVISIONING_STATES,
+    ...HOST_REGISTERING_STATES,
```

Comment:
`HOST_REGISTERING_STATES` and `HOST_PROVISIONING_STATES` are not needed as those are a subset of `HOST_PROGRESS_STATES`

Reply:
good to know, removed redundant states